_layouts/post.html

```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+css: ["post.css"]
+js: ["post.js"]
+---
+<div id="post-info">
+  <h3>{{page.title}}</h3>
+  <ul class="collapsible hoverable" data-collapsible="accordion">
+    <li>
+      <div class="collapsible-header">
+        <span>
+          <i class="material-icons tooltipped" data-position="left" data-delay="30" data-tooltip="Date">date_range</i>
+          {{page.date | date: "%d/%m/%Y %H:%M"}}
+          <i id="indicate" class="right material-icons tooltipped" data-position="left" data-delay="30" data-tooltip="Show extra info">info</i>
+        </span>
+      </div>
+      <div class="collapsible-body">
+        <span>
+          <i class="material-icons tooltipped" data-position="left" data-delay="30" data-tooltip="Categories">sort</i>
+          {% for cat in page.categories %}
+          {% capture test %}{{cat | slice: 0}}{% endcapture %}
+          {% capture testup %}{{cat | slice: 0 | upcase}}{% endcapture %}
+          <a href="{{site.baseurl}}/categories#{{cat | slugify}}{% if test == testup %}_cap{% endif %}" target="_blank"><div class="chip">{{cat}}</div></a>
+          {% endfor %}
+        </span>
+        <span>
+          <i class="material-icons tooltipped" data-position="left" data-delay="30" data-tooltip="Tags">label</i>
+          {% for tag in page.tags %}
+          {% capture test %}{{tag | slice: 0}}{% endcapture %}
+          {% capture testup %}{{tag | slice: 0 | upcase}}{% endcapture %}
+          <a href="{{site.baseurl}}/tags#{{tag | slugify}}{% if test == testup %}_cap{% endif %}" target="_blank"><div class="chip">{{tag}}</div></a>
+          {% endfor %}
+        </span>
+      </div>
+    </li>
+  </ul>
+</div>
+<div class="divider"></div>
+<div class="row">
+  <div class="col s12">
+    {{ content }}
+  </div>
+</div>
+{% if site.disqus_shortname and page.comments %}
```

Comment:
I'm not sure we'll ever use this so perhaps remove? Or is it easier to make fewer changes from the theme we're basing this on so we can sync up again in future?

Reply:
Good point. Removed the file.